templates/404.html

```diff
@@ -23,15 +22,38 @@ <h1><a id="home-link" href="/">TMPNOTES 🔥️</a></h1>
         <h2>Nothing to see here 👀️</h2>
     </div>
 </body>
-
-<div class="container" style="border:1px solid white;">
+<div class="container" id="footer-container">
     <div class="footer">
         <div>
             <p>ABOUT TMPNOTES</p>
-            <p>Thanks for using TMPNOTES! TMPNOTES is an open source project that allows notes to be passed simply and securely.</p>
-            <p>Like the concept of TMPNOTES but don't trust those hosting it? Feel free to host it yourself - <a href="https://github.com/soraro/tmpnotes#run-it-yourself">self hosting instructions</a>.</p>
-            <p>Enjoy using TMPNOTES? Give our GitHub repo a star! <a class="github-button" href="https://github.com/soraro/tmpnotes" data-icon="octicon-star" aria-label="Star soraro/tmpnotes on GitHub">Star</a></p>
-            <p>Running into bugs? Missing a feature you want? Create an issue! <a class="github-button" href="https://github.com/soraro/tmpnotes/issues" data-icon="octicon-issue-opened" aria-label="Issue soraro/tmpnotes on GitHub">Issue</a></p>
+            <p>Thanks for using TMPNOTES! TMPNOTES is an open source project that allows notes to be passed simply and
+                securely.</p>
+            <p>Like the concept of TMPNOTES but don't trust those hosting it? Feel free to host it yourself - <a
+                    href="https://github.com/soraro/tmpnotes#run-it-yourself">self hosting instructions</a>.</p>
+            <p>Enjoy using TMPNOTES? Give our GitHub repo a star!
+                <a href="https://github.com/soraro/tmpnotes">
+                    <button type="button" class="btn btn-primary" id="gh-button-star">
+                        <svg xmlns="http://www.w3.org/2000/svg" width="12" height="12" fill="currentColor"
+                            class="bi bi-star-fill" viewBox="0 0 16 16">
+                            <path
+                                d="M3.612 15.443c-.386.198-.824-.149-.746-.592l.83-4.73L.173 6.765c-.329-.314-.158-.888.283-.95l4.898-.696L7.538.792c.197-.39.73-.39.927 0l2.184 4.327 4.898.696c.441.062.612.636.282.95l-3.522 3.356.83 4.73c.078.443-.36.79-.746.592L8 13.187l-4.389 2.256z">
```

Comment:
where did you get this from?

Reply:
https://icons.getbootstrap.com/icons/github/

and 

https://icons.getbootstrap.com/icons/star-fill/

I know it's not quite the same as before, but I actually think it gets pretty close!

Reply:
👍